.travis.yml

```diff
@@ -12,7 +12,17 @@ env:
   - TOXENV=py33-1.8
   - TOXENV=py34-1.8
   - TOXENV=py34-1.9
+  - TOXENV=py34-1.10
+  - TOXENV=py34-1.11
   - TOXENV=py34-master
+  - TOXENV=py35-1.9
+  - TOXENV=py35-1.10
+  - TOXENV=py35-1.11
```

Comment:
looks like 1.11 test is failing for 3.4 at least: https://travis-ci.org/charettes/django-colorful/jobs/220687623

I don't have time to fix that, I merely wanted to see if this package is ready for 1.11.